src/NuGet.Clients/NuGet.CommandLine/xlf/NuGetCommand.es.xlf

```diff
@@ -993,7 +993,7 @@ nuget sign MyPackage.nupkg -CertificateFingerprint certificate_fingerprint -Outp
       </trans-unit>
       <trans-unit id="SignCommandNoCertificateException">
         <source>No certificate was provided. For a list of accepted ways to provide a certificate, please visit https://docs.nuget.org/docs/reference/command-line-reference</source>
-        <target state="new">No certificate was provided. For a list of accepted ways to provide a certificate, please visit https://docs.nuget.org/docs/reference/command-line-reference</target>
+        <target state="translated">No se encontró el archivo de certificado "{0}". Para obtener una lista de las formas aceptadas de proporcionar un certificado, visite https://docs.nuget.org/docs/reference/command-line-reference</target>
```

Comment:
This isn't correct...there's no `"{0}"` in the original source string, and so it'll cause an issue in the build, not to mention the localization itself. Any idea where this happens to be coming from?

Reply:
This was in the original translations...and was causing build issues then...I wonder where it's coming from.

Reply:
If I'm looking at the right spot, this string has not changed in 8 years and does not have `{0}`.
Is the loc tooling adding that placeholder somehow?
https://github.com/NuGet/NuGet.Client/blame/546b0dc025190ab9cf744ad4be544c23042688c1/src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx#L750C51-L750C51

Reply:
I think this may be coming from the old localization. It's incorrect here:

https://github.com/NuGet/NuGet.Build.Localization/blob/0856dcd9e3b6f2a90ccdeffc718c691e5660026f/localize/ESN/15/NuGet.exe.lcl#L1542-L1550

I am not sure about how to keep this from happening. Do we have a contact on the loc team?

Reply:
Created a ticket with the loc team https://ceapex.visualstudio.com/CEINTL/_workitems/edit/1077270